modules/navier_stokes/test/tests/finite_volume/wcns/materials/2d-transient.i

```diff
@@ -222,6 +223,10 @@ inlet_v = 0.001
     T_fluid = 'T'
     speed = 'velocity_norm'
 
+    # even though we provide rho from the parameters, we
+    # want to get rho from the fluid properties
+    force_define_density = true
```

Comment:
Yea I don't get how this works haha. Why is there a need for this parameter? Should we just always create `rho` from `GeneralFunctorFluidProps`? I guess not for incompressible cases?

Reply:
yeah not for incompressible. 
But it really depends, if there is a fp, you could be wanting to use the rho from it (simpleFluidProp is not bad for incompressible)

I see this as a workaround